cli/client.go

```diff
@@ -572,6 +574,16 @@ var clientRetrieveCmd = &cli.Command{
 		if minerStrAddr == "" { // Local discovery
 			offers, err := fapi.ClientFindData(ctx, file, pieceCid)
 
+			var cleaned []api.QueryOffer
+			// filter out offers that errored
+			for _, o := range offers {
+				if o.Err != "" {
```

Comment:
Am I crazy, or does this filter *in* offer that errored and exclude the rest?

Reply:
oh hush.